packages/components/src/DataTable/DataTable.test.tsx

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { DataTable } from ".";
+import { Pagination, Sorting } from "./types";
+
+const data = [
+  {
+    name: "Eddard",
+    house: "Stark",
+    region: "North",
+    sigil: "Direwolf",
+    isAlive: "No",
+  },
+  {
+    name: "Catelyn",
+    house: "Stark",
+    region: "North",
+    sigil: "Direwolf",
+    isAlive: "No",
+  },
+  {
+    name: "Jon Snow",
+    house: "Stark",
+    region: "North",
+    sigil: "Direwolf",
+    isAlive: "Yes",
```

Comment:
Should this say "Resurrected" instead of "Yes" 😂

Reply:
hahahahaha 😂

Reply:
This was such a cool suggestion. Gave me the idea to add an `accessorFn`example 🤌

Reply:
✅